apps/condo/domains/common/utils/import/index.js

```diff
@@ -0,0 +1,5 @@
+const importDateUtils = require('@condo/domains/common/utils/import/importDate.utils')
+
+module.exports = {
+    ...importDateUtils,
```

Comment:
Its' a bad practice to have dynamic exports, since you can export something wrong from your module, and it will be chained here.

Good practice to explicitly specify, what you want others to use